### PR TITLE
Add one more star in PDOStatementHandle's PHPDoc

### PR DESCRIPTION
**Pull request by szepeviktor**
@sminnee

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,7 +56,9 @@
     "require-dev": {
         "phpunit/phpunit": "^5.7",
         "sminnee/phpunit-mock-objects": "^3.4.5",
-        "silverstripe/versioned": "^1"
+        "silverstripe/versioned": "^1",
+        "symbiote/silverstripe-phpstan": "dev-phpstan011",
+        "phpstan/phpstan-shim": "^0.11"
```
Comment (szepeviktor) [inline composer.json]: @sminnee Please use the modern PHAR distro: `        "phpstan/phpstan": "^0.12"`

Comment (sminnee) [reply composer.json]: good point

Comment (szepeviktor) [reply composer.json]: @sminnee Friendly ping 🏓

**Comment by szepeviktor** [comment]
Oh nooo!
PDO was removed in https://github.com/silverstripe/silverstripe-framework/pull/10499

**Pull request closed without merging**